src/compiler/types.ts

```diff
@@ -3432,7 +3432,7 @@ namespace ts {
         ExportStar              = 1 << 23,  // Export * declaration
         Optional                = 1 << 24,  // Optional property
         Transient               = 1 << 25,  // Transient symbol (created during type check)
-        JSContainer             = 1 << 26,  // Contains Javascript special declarations
+        Assignment              = 1 << 26,  // Assignment treated as declaration (eg `this.prop = 1`)

```

Comment:
The comments make it sound like the meaning changed -- previously this would be the symbol for the thing you're assigning to properties of, now it sounds like the symbol for a particular assignment?

Reply:
Yes, I just merged #26690, which puts this flag on *all* assignment declarations, not just ones that are themselves containers. I decided to put all the renames in this PR, though.